js/views/site.es6.js

```diff
@@ -16,6 +18,16 @@ function Site (ops) {
       [this.$whitelisttoggle, 'click', this._whitelistClick]
     ]);
 
+    // FIXME should be moved to the right place --- model?
```

Comment:
Ok. Looks good then. Can you switch the PR to the beta branch so we can merge it?